.github/workflows/unit_test_and_quality.yml

```diff
@@ -62,11 +62,11 @@ jobs:
 
       # - name: Run Tests with Coverage
       #   run: make testcov
-      
+
```

Comment:
```suggestion
```

I originally left these, and since we're modifying this file, let's just remove them.